### ISSUE TITLE
Heel squeeze test positive

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11190]
**Preferred term label:**
Heel squeeze test positive

**Synonyms**

**Definition (free text, please give PubMed ID)**
Compression of the medial and lateral side of the calcaneus/heel area elicits pain

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Sever's disease
Calcaneus fractures

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001350
- label: Positive heel squeeze test
- parents: [Abnormal ankle physiology (HP:0034674)](https://hpo.jax.org/app/browse/term/HP:0034674)
- pmid: PMID:24072363|PMID:39430116
- def: The squeeze test is the most common examination for diagnosing calcaneal apophysitis (Sever disease) and is conducted by compressing the medial and lateral aspects of the calcaneus. The test is considered positive (abnormal) if squeezing elicits pain.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11190
- synonyms: Positive calcaneal squeeze test(NONE)
